src/code/dialogs/checklist.js

```diff
@@ -233,18 +238,23 @@ const OperationChecklistDialog = WDialog.extend({
           const a = L.DomUtil.create("a", null, cell);
           a.href = "#";
           a.textContent = wX(value);
-          L.DomEvent.on(cell, "click", (ev) => {
-            L.DomEvent.stop(ev);
-            const sd = new StateDialog({
-              target: thing,
-              opID: operation.ID,
+          // XXX: should be possible with atomic api call
+          if (canWrite) {
```

Comment:
we can check assignment here, and display atomic options if assigned to WasabeeMe.gid

Reply:
but not today, for 0.21 or later